app/models/concerns/manifest_concern.rb

```diff
@@ -5,4 +5,9 @@ module ManifestConcern
   def manifest
     fetch('iiif_manifest_url_ssi', nil)
   end
+
+  def exhibit_specific_manifest(custom_manifest_pattern)
+    return manifest if custom_manifest_pattern.blank?
+    custom_manifest_pattern.gsub('{id}', fetch('id', nil))
```

Comment:
I'm totally 👍 on this approach.  Just wanted to bring up an alternative for consideration (I don't think we know enough to know what is the right approach, but if this makes its way to Spotlight it might be worth thinking through).

If we use the `%{id}` syntax then we can use `String%` (e.g. `custom_manifest_pattern % { id: fetch('id', nil) }`).

The benefit of this being that we could allow an application to configure the set of document keys they will allow to be interpolated into a manifest.

Reply:
Thanks for that feedback. Yeah I was also thinking that any Solr field could be used in the pattern too, but didn't want to go that far into it until we needed to.

Great suggestion. 🆗 to keep this approach for this pr?

Reply:
You should be able to use `first('id')` rather than `fetch('id', nil)`. See https://github.com/projectblacklight/blacklight/blob/dee8d794125306ec8d4ab834a6a45bcf9671c791/app/models/concerns/blacklight/document.rb#L89

Reply:
Nice catch, I've updated to use your suggestion.